stories/Rate.stories.js

```diff
@@ -1,15 +1,19 @@
 import React, { Component } from "react";
 import Rate from "../src/components/rating/Rate";
-import { boolean } from "@storybook/addon-knobs";
+import { boolean, number } from "@storybook/addon-knobs";
 
 class RatingTest extends Component {
     render() {
         return (
-            <Rate
-                label="Rating"
-                value={4.5}
-                readOnly={boolean("ReadOnly", false)}
-            />
+            <React.Fragment>
```

Comment:
Since the `<React.Fragment>` only has one child element, `Rate`, I don't think we actually need `<React.Fragment>` here.  It doesn't hurt to have it either though

Reply:
I was playing with rendering multiple Rating widgets with different props for testing purpose. I removed additional widgets but forgot to remove the `<React.Fragment>`.